# Linear relative measurement msg, cartesian coordinates
std_msgs/Header header
string robot_measured
float64 x		# m
float64 y	   	# m
float64 z  	    # m
